packages/bits/src/lib/wizard-v2/wizard.directive.ts

```diff
@@ -135,20 +124,25 @@ export class WizardDirective extends CdkStepper implements OnChanges, AfterConte
         super.ngOnDestroy();
     }
 
-    public getStepState(i: number, state: StepState = STEP_STATE.NUMBER): StepState {
+    public getStepState(i: number, stepState: StepState = STEP_STATE.NUMBER): StepState {
```

Comment:
I think I've asked this before, but I'm not sure why we pass in the stepState here.

Reply:
this is a leftover. thanks for capturing